test/views/partials/conversationInboxSpec.scala

```diff
@@ -22,14 +22,21 @@ import views.viewmodels.ConversationInbox
 
 import scala.util.{ Success, Try }
 
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
 class conversationInboxSpec extends TemplateUnitSpec[ConversationInbox] {
 
   "A conversation inbox template" must {
     "have all mandatory information" in {
-      render(ConversationInbox("cds-frontend", "test", List.empty)) match {
+      render(ConversationInbox("cds-frontend", "test", 5, 1, List.empty)) match {
         case Success(component) => {
-          val res = component.select("h1")
-          res.text mustBe "test"
+          val pageContent = component.body
+          pageContent must include("""<h1 class="govuk-heading-xl">test</h1>""")
+          pageContent must include("""<span class="govuk-visually-hidden"> 1 unread</span>""")
+          pageContent must include("""<span  class="govuk-visually-hidden"> 5 in total </span>""")
```

Comment:
might be better off to use Jsoup and parse the HTML document to do the asserts on specific elements because doing asserts on string literals will probably render your tests to be frail over time. Don't worry though, I'll refactor those on my next commit.